nets/DevNetSpec.ts

```diff
@@ -41,11 +42,15 @@ export abstract class DevNetSpec extends NetSpec {
   }
 
   async rawChainSpecPath(signal: AbortSignal, devnetTempDir: string) {
-    return createRawChainSpec(
-      this.tempDir(devnetTempDir),
-      await this.binary(signal),
-      this.chain,
-    )
+    const binary = await this.binary(signal)
+    if (!this.#rawChainSpecPath) {
+      this.#rawChainSpecPath = createRawChainSpec(
+        this.tempDir(devnetTempDir),
+        binary,
```

Comment:
awaiting on this line does not prevent the race condition because the `!this.#rawChainSpecPath` will not be set until `await this.binary(signal)` resolves